app/_data/tables/support/gateway/versions/33.yml

```
@@ -0,0 +1,95 @@
+{% include_relative_once _data/tables/support/gateway/packages.yml %}
+{% include_relative_once _data/tables/support/gateway/third-party.yml %}
+{% include_relative_once _data/tables/support/gateway/browsers.yml %}
+
+lts: false
+distributions:
+  - <<: *alpine
+    docker: true
+  - <<: *amazonlinux2
+    docker: true
+    arm: true
+  - <<: *amazonlinux2023
+    docker: true
+    arm: true
+  - *centos7
+  - *debian10
+  - <<: *debian11
+    docker: true
+  - *rhel7
+  - <<: *rhel8
+    docker: true
+    fips: true
+  - <<: *ubuntu1804
+    arm: false
+    docker: false
+  - <<: *ubuntu2004
+    arm: false
+    docker: false
+    fips: true
+  - <<: *ubuntu2204
+    arm: true
+    docker: true
+    fips: true
+
+third-party:
+  datastore:
+    - <<: *postgres
+      versions:
+        - Amazon Aurora
```

```suggestion
      versions:
        - 15
        - 14
        - 13
        - 12
        - 11
        - 10
        - 9
        - Amazon RDS
        - Amazon Aurora
```
Apparently you have to add the whole list. Didn't realize that.